### ISSUE TITLE
Manage multiple policies

### ISSUE THREAD

**Issue by szamarripa** [MusesProject/MusesServer#42]
Proposal:

	I thought about the issue why we are seeing just one (random) pop up for the security_property change violations (which are three: screen timeout, accessibility and antivirus) and I think I found the problem. The issue is, that for a single request I can get multiple policies, which come sequentially. When I receive the first policy, I ask the decision maker again to make a decision, and I retrieve one for this policy. But after the first policy update, the request (which holds the action object and the properties) is deleted, because a single requests expects a single response, as I don’t know that there can be multiple responses and if there are multiple, I cannot know how many I have to expect. 

So how to handle it?  It would require two major changes:
1. If a request results in multiple policies, send all policies in a single JSON response.
2. The decision maker should return and Array of Decisions, instead of a single one, because we can have multiple decisions for a single user action. 

Minor Changes:
1. Update JSON functionality on the client and server
 
When we follow this solution, first, all policies will be in the database before I ask the decision maker. Second, we are able to handle multiple decision per action type.

**Comment by szamarripa** [comment]
This is a nice improvement to be done, but it should be done as pure development. I propose to create a new Milestone toward product and assign it to it.
